recipes/install-ca-webapp-packages.rb

```diff
@@ -0,0 +1,17 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-mh-base-packages
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+include_recipe "mh-opsworks-recipes::update-package-repo"
+
+%w|python-dev
+python-virtualenv
+python-pip
+supervisor
+libpq-dev
+libffi-dev
+nginx|.each do |package_name|
+  install_package(package_name)
```

Comment:
you can pass a string of packages to `install_package` and it'll do them all at once, which is slightly faster. Sorry for the array antipattern I used to use.